src/keylessBackup/hooks.ts

```diff
@@ -0,0 +1,150 @@
+import { useRef, useState } from 'react'
+import { useAsync } from 'react-async-hook'
+import { Platform } from 'react-native'
+import DeviceInfo from 'react-native-device-info'
+import { useDispatch } from 'react-redux'
+import { showError } from 'src/alert/actions'
+import { KeylessBackupEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { ErrorMessages } from 'src/app/ErrorMessages'
+import { valoraKeyshareIssued } from 'src/keylessBackup/slice'
+import { KeylessBackupFlow } from 'src/keylessBackup/types'
+import Logger from 'src/utils/Logger'
+import { PhoneNumberVerificationStatus } from 'src/verify/hooks'
+import networkConfig from 'src/web3/networkConfig'
+
+const TAG = 'keylessBackup/hooks'
+
+export function useVerifyPhoneNumber(phoneNumber: string, keylessBackupFlow: KeylessBackupFlow) {
```

Comment:
This is copied from https://github.com/valora-inc/wallet/blob/0c04f6beb7fffa84988839d15bb078e3bb7d44c4/src/verify/hooks.ts#L36
and adjust to call the CAB endpoints. 
There are no specific unit tests for this. Hooks need a wrapper component for testing, it was easier to just test from the screen component similar to the CPV screen tests